manual/manual/Makefile

```diff
@@ -76,7 +76,7 @@ html: files
 	         -I ../tutorials -I ../../styles -I ../texstuff manual.hva \
 	         -e macros.tex ../manual.tex ; \
 	${HACHA} -tocter manual.html ; \
-
+	cat ../manual.css >> manual.css
```

Comment:
This looks like sort of a hack; what is the compatibility story for downstream ocamldoc users? I thought that ocamldoc would also provide a default style file that they can use, and we can adapt on our end, without having to change Hevea as well.